src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -66,7 +67,99 @@
 import static org.apache.cassandra.io.sstable.format.SSTableFormat.Components.DATA;
 
 /**
- * Legacy bigtable format
+ * Legacy bigtable format. Components and approx lifecycle:
```

Comment:
can we use full for of this? - `approximate`? Later you have word "approximation".